Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM python:3.7.7-slim
+
+# Create virtualenv and add to path.
+ENV VIRTUAL_ENV=/opt/venv
+RUN python3 -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+# System dependencies
+RUN apt-get update && apt-get install -y build-essential && rm -rf /var/lib/apt/lists/*
+
+WORKDIR /opt/app
```

Comment:
Is it possible to mount relative to this `WORKDIR`? It feels clunky to have to specify it again when running tests:
```
$ docker run --rm -v `pwd`:/opt/app xocto
```

Reply:
I've not seen it done, and I can't find a reference to it online, I'm afraid :-( would be nice though!